### ISSUE TITLE
[Bug] Large Iron Double Doors are destroyable by hand like a torch

### ISSUE THREAD

**Issue by ExDomino** [Ordinastie/MalisisDoors#253]
Hello,

There is a problem with large double doors, especialy those in iron, that can be destroyed by hand with a single hit. Can you reinforce their durability please? Make those a bit more resistant than vanilla iron doors?

Thank you in advance.

Same issue I think about large wood doors and rusty iron doors.

**Comment by ate0ate** [comment]
Yeah, I gotta agree here. I love the larger 3x3 doors but I don't know how many times I've broken them empty handed with one mouse click, let alone whenever hitting a mob anywhere near it. And you are correct, it seems to be all of the larger doors.

**Comment by Draskh** [comment]
I have the same issue. That should be fixed.

**Comment by ExDomino** [comment]
Sorry, but I think this mod is abandonned.

**Comment by Draskh** [comment]
I don't think so. Ordinastie is still active.

**Comment by ExDomino** [comment]
But he does no longer watch his bug tracker then.

**Comment by Draskh** [comment]
I think he would be busy.

**Comment by ExDomino** [comment]
Busy since 1 feb 2018.

**Comment by Draskh** [comment]
I can't tell you. :D

**Comment by ExDomino** [comment]
Still not fixed ...

**Comment by Draskh** [comment]
I have a fix made by my own with CraftTweaker. Inserting this into a script should fix the bug. 

#Fixing Malisis Doors
val ironDoor = <malisisdoors:big_door_iron_3x3>;
ironDoor.hardness = 5.0;

I also discovered that some other doors have the same issue. However they all can be fixed with this.
